cellcom/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "cellcom"
```

Comment:
can we name it `orb-cellcom`? the `orb-` prefix is useful for consistency and to disambiguate between our crates and crates.io

Reply:
As it's touching WiFi and LTE, maybe we can even name it `orb-location` or `orb-geolocation`? The second one sounds cool and is obviously descriptive